testing/src/main/java/io/stargate/it/http/graphql/graphqlfirst/SchemaDeploymentTest.java

```diff
@@ -386,4 +387,52 @@ public void deploySchemaErrorNoStacktrace(@TestKeyspace CqlIdentifier keyspaceId
     // to the spec's format to avoid that:
     assertThat(schemaError).doesNotContainKey("stackTrace");
   }
+
+  @Test
+  @DisplayName("Should deploy schema and get recent schema with logs")
+  public void deploySchemaAndGetRecentSchemaWithLogs(
+      @TestKeyspace CqlIdentifier keyspaceId, CqlSession session) {
+    // given
+    String expectedMessage =
+        "User: using id as the partition key, because it has type ID and no other fields are annotated";
+    String expectedCategory = "Info";
+    String expectedLocations = "1:1";
+    String keyspace = keyspaceId.asInternal();
+
+    // when
+    CLIENT.deploySchema(keyspace, SCHEMA_CONTENTS);
+
+    // then
+    Row row =
+        session
+            .execute(
+                "select * from stargate_graphql.schema_source wHERE keyspace_name = ?", keyspace)
+            .one();
+    assertThat(row).isNotNull();
+    TupleValue log = row.getList("logs", TupleValue.class).get(0);
+    assertThat(log.getString(0)).isEqualTo(expectedMessage);
+    assertThat(log.getString(1)).isEqualTo(expectedCategory);
+    assertThat(log.getString(2)).isEqualTo(expectedLocations);
+
+    // when
+    Object response =
+        CLIENT.executeSchemaQuery(
+            String.format(
+                "{\n"
+                    + "  schemas(keyspace: \"%s\") {\n"
+                    + "    version\n"
+                    + "    contents\n"
+                    + "    logs { message, category, locations } \n"
```

Comment:
I think that we should not propagate the tuple to callers, but encapsulate that using a dedicated type (https://github.com/stargate/stargate/pull/998/files#diff-c7af9322a34dce3e3db2ca1af6aad38ddb19a2d8f831a2c67e2ea0f636daebf6R55). It seems a lot more user friendly, than letting our users handle tuples. @olim7t do you agree?

Reply:
We already have such a type in the schema: `AdminSchemaBuilder.LOG_TYPE`.

Reply:
The existing `LOG` type uses enum for category and custom type for location:
```
   newObject()          .name("Log")          .description("A log emitted by a deploy mutation.")          .field(newFieldDefinition().name("message").type(nonNull(GraphQLString)).build())          .field(newFieldDefinition().name("category").type(nonNull(LOG_CATEGORY_ENUM)).build())          .field(newFieldDefinition().name("locations").type(list(LOCATION_TYPE)).build())          .build();
```
As we discussed in the ticket, we wanted to have a simpler solution here (`tuple<string,string,string>`). Therefore I think we need this new type.